<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="102908" started-at="2016-05-31T10:05:21Z" finished-at="2016-05-31T10:07:04Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="102908" started-at="2016-05-31T10:05:21Z" finished-at="2016-05-31T10:07:04Z">
      <class name="TestCases.validateFooterLinks">
        <test-method status="PASS" signature="launchWebPage()[pri:0, instance:TestCases.validateFooterLinks@5ae63ade]" name="launchWebPage" is-config="true" duration-ms="4995" started-at="2016-05-31T15:35:21Z" finished-at="2016-05-31T15:35:26Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- launchWebPage -->
        <test-method status="FAIL" signature="verifyFooterLinks()[pri:0, instance:TestCases.validateFooterLinks@5ae63ade]" name="verifyFooterLinks" duration-ms="97600" started-at="2016-05-31T15:35:26Z" finished-at="2016-05-31T15:37:04Z">
          <exception class="java.lang.IndexOutOfBoundsException">
            <message>
              <![CDATA[Index: 23, Size: 23]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.IndexOutOfBoundsException: Index: 23, Size: 23
	at java.util.ArrayList.rangeCheck(Unknown Source)
	at java.util.ArrayList.get(Unknown Source)
	at TestCases.validateFooterLinks.verifyFooterLinks(validateFooterLinks.java:43)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:646)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:823)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1131)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:778)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1225)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1150)
	at org.testng.TestNG.runSuites(TestNG.java:1075)
	at org.testng.TestNG.run(TestNG.java:1047)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:126)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:137)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:58)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.IndexOutOfBoundsException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyFooterLinks -->
        <test-method status="PASS" signature="closeBrowser()[pri:0, instance:TestCases.validateFooterLinks@5ae63ade]" name="closeBrowser" is-config="true" duration-ms="281" started-at="2016-05-31T15:37:04Z" finished-at="2016-05-31T15:37:04Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- closeBrowser -->
      </class> <!-- TestCases.validateFooterLinks -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
